contrib/epee/include/net/http_server_impl_base.h

```diff
@@ -31,7 +31,7 @@
 
 
 #include <boost/thread.hpp>
-#include <boost/bind.hpp> 
+#include <boost/bind/bind.hpp>
```

Comment:
boost bind still ?

Reply:
There are 88 occurrences of `boost::bind`, outside of the usage in `src/simplewallet/simplewallet.cpp`. There are 120 occurrences in just that file. I decided to only update the places that used `_1`, etc, to keep this smaller and readable. Updating the remaining seemed like it could be done in a subsequent patch, but I can do a `sed` call right now and re-test the compilations ... ? With or without updating `simplewallet.cpp`.

Reply:
Update - only 39 of 88 are from in `src/` but outside of `src/simplewallet/simplewallet.cpp`. Meaning, some of the stuff in `contrib` is dead unused/code I think, so I may not need to update all 88 occurrences. Its difficult to know how many without trial and error though.

Reply:
Makes sense. I'm in favour of leaving it as is if it doesn't need fixing :)